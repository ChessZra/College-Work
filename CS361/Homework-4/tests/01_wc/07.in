wc
there are     multiple
spaces   between   these
words    words  words
