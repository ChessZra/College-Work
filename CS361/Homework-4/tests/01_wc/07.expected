cs361sh> Num Bytes: 70
Num Lines: 3
Num Words: 9
cs361sh> 
